### PR TITLE
Added metadata term validation.

### PR DESCRIPTION
**Pull request by alesstimec**

### CODE REVIEW

File: meta.go [hunk 1]
```diff
@@ -238,6 +238,22 @@ func parseStringList(list interface{}) []string {
...
+func checkTerm(s string) error {
+	if strings.Contains(s, " ") {
+		return errors.New("term may not contain spaces: " + s)
+	}
+	tokens := strings.Split(s, "/")
+	if len(tokens) != 2 {
+		return errors.New("unknown term format: " + s)
+	}
+
+	_, err := strconv.Atoi(tokens[1])
```
Comment (cmars) [inline meta.go]: I think `tokens[0]` ought to match a regexp like `^[a-z]+([a-z0-9-]+)?$`:
- Must start with a lowercase letter
- Followed by any combination of lowercase letters a-z, digits and dashes

File: meta.go [hunk 2]
```diff
@@ -238,6 +238,29 @@ func parseStringList(list interface{}) []string {
 	return result
 }
 
+var termNameRE = regexp.MustCompile("^[a-z]+([a-z0-9-]+)?$")
+
+func checkTerm(s string) error {
+	if strings.Contains(s, " ") {
```
Comment (tasdomas) [inline meta.go]: This could be a five line function with a proper regular expression.

**Comment by tasdomas** [comment]
This looks fine as it is, but it's probably one of the places where it's ok to use regex.

**Comment by cmars** [comment]
LGTM, thanks!

**Comment by alesstimec** [comment]
$$merge$$ :shipit:

**Pull request merged**